Dockerfile

```diff
@@ -1,4 +1,22 @@
-FROM python:3.8
+FROM python:3.8 as build-python
+
+ENV VIRTUAL_ENV=/opt/venv \
+    KOBOCAT_SRC_DIR=/srv/src/kobocat \
+    TMP_DIR=/srv/tmp
+
+# Copy KoBoCAT directory
+COPY . "${KOBOCAT_SRC_DIR}"
+
+# Install `pip` packages
+RUN python3 -m venv "$VIRTUAL_ENV"
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+RUN pip install  --quiet --upgrade pip && \
+    pip install  --quiet pip-tools
+COPY ./dependencies/pip/prod.txt "${TMP_DIR}/pip_dependencies.txt"
```

Comment:
I think that this file needs to be copied too into the next stage, otherwise we cannot install `pip-sync` dev dependencies on start-up.

```
kobocat_1         | KoBoCAT initialization complete.
kobocat_1         | diff: /srv/tmp/pip_dependencies.txt: No such file or directory
```

https://github.com/kobotoolbox/kobocat/blob/beta/docker/run_uwsgi.bash#L31-L45

Reply:
Should be addressed, `DIFF=$(diff "${KOBOCAT_SRC_DIR}/dependencies/pip/dev.txt" "${TMP_DIR}/pip_dependencies.txt")` should now run.